import-service/src/importFileParser.js

```diff
@@ -0,0 +1,45 @@
+const AWS = require("aws-sdk");
+const csv = require("csv-parser");
+const { FOLDER_UPLOAD, FOLDER_PARSED } = require('./configs');
+
+module.exports = async (event) => {
+  console.log('importFileParser input: ',JSON.stringify(event))
+
+  const s3 = new AWS.S3({ signatureVersion: "v4" });
```

Comment:
s3 instance could be outside of the lambda and reused where u need e.g. in importProductsFile

Reply:
As I know each lambda invocation is performed on separate env and exposing initialization of AWS module will not be reusable. i.e. importFileParser will init the AWS and importFileParser also will init the AWS separately
also s3 copyObject and deleteObject used only in that function there is no need to reuse it.